application/controllers/Qso.php

```diff
@@ -431,9 +431,11 @@ function delete_ajax() {
 
 	function band_to_freq($band, $mode) {
 
-		$this->load->library('frequency');
+		if ($band != null and $band != 'null') {
```

Comment:
works with PHP8.3. but: wouldn't

`if (($band ?? '') != '') {`

be better, and more failsafe? or is the string "null" coming from JS?

Reply:
> is the string "null" coming from JS?

Yepp. See the screenshot of the request above.

Reply:
roger. in that case: approve